src/config.rs

```diff
@@ -269,14 +269,22 @@ impl Config {
         /// and returns it
```

Comment:
Give some sample input and sample outputs also for this function.

Reply:
Sample outputs:
```markdown
bar/index.ftd -> /bar/
index.ftd -> /
dotcom/src/header.ftd -> /dotcom/src/header/
fpm.dev/doc.ftd -> /fpm.dev/doc/
```
I will mention some sample outputs in the code itself.

Reply:
Look at more corner cases, eg /bar/index.ftd, what happens to this? Or foo.png? foo.md?